### ISSUE TITLE
Figure output height too large

### ISSUE THREAD

**Issue by FieteO** [bupaverse/processmapR#35]
The plots produced by the `process_map` function have too much vertical padding for my taste. It would be great if the height could be modified to take up less space (in an rmarkdown notebook).

I tried modifying the size using knittr chunk options but that didn't really work and would be unwieldy as well. #23 may be related to this as well. 
## Current rendering of plot
![image](https://user-images.githubusercontent.com/41323592/147923032-b48590d7-0c5d-47c7-a315-d24ba4f07664.png)
 ## Desired rendering of plot
![image](https://user-images.githubusercontent.com/41323592/147924349-aab9cf65-f09f-41cf-abf3-40d1a377c001.png)

**Comment by gertjanssenswillen** [comment]
One very coarse work-around in to change the direction from left-right to top-bottom. This can be done with the argument rankdir = "TB". 

Other ways to configure the width are currently not available.

**Comment by gertjanssenswillen** [comment]
Does this help: https://stackoverflow.com/questions/61794098/diagrammer-and-r-markdown-pdf-remove-space-around-diagram

**Comment by FieteO** [comment]
Not really, but thanks for investigating. Do I understand it right, that this is more of a `DiagrammeR` issue, then a `processmapR` one?

**Comment by gertjanssenswillen** [comment]
Yes, DiagrammeR is responsible for the rendering, and so far I have not found any parameters to fix it. Also looking into alternative packages for showing the graphs atm. I'll keep you updated
